specification/communication/data-plane/OperatorConnect/preview/2022-04-03-preview/operatorconnect.json

```diff
@@ -0,0 +1,495 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Operator Connect",
+    "version": "2022-04-03-preview"
+  },
```

Comment:
You should add `securityDefinitions` and a global `security` to the API definition.  Typically this is an "oauth2" security scheme that applies to operations.

Reply:
Hi @mikekistler i have added the secrutityDefinition and global security node. Should we merge this PR even if it's just for private preview?
Also, what should we do with the other PR in the PR repo azure-rest-api-specs-pr? Do we abandon that one?
https://github.com/Azure/azure-rest-api-specs-pr/pull/6247
Please let us know what is desired practice.

Reply:
It looks like this is good to merge. @tjprescott will you to the honors?

I think you can just close the PR in azure-rest-api-specs-pr ... maybe with a comment that points to this one.

Reply:
Hi @mikekistler and @tjprescott can we merge this please?

Reply:
Pinging the thread, can we merge this? cc @mikekistler and @tjprescott

Reply:
hi @JeffreyRichter @mikekistler and @tjprescott we are unsure what we need to do to proceed to merge this PR.
We went through review, and got approval. Is there anything else pending or it can be merged? We dont have rights to merge it so it has to be done by someone from reviewiers.